lib/addons/p5.dom.js

```diff
@@ -21,7 +21,7 @@
  *
  * @module p5.dom
  * @submodule p5.dom
- * @for p5.dom
+ * @for p5
```

Comment:
I'm not sure I understand what this does, can you explain where else this is applied and why it was changed?

Reply:
sure, the `@for` attribute specifies the class that the following methods are members of. if you look in the `data.json` all these dom methods are listed as being in the `p5.dom` class. it didn't matter before that this was wrong because it wasn't used anywhere, but now that the `lookupParamDoc` method is looking up docs my class name, this needs to be fixed.